src/sources/aws_kinesis_firehose/handlers.rs

```diff
@@ -242,7 +243,7 @@ fn is_gzip(data: &[u8]) -> bool {
     // deflate algorithm.
     //
     // Reference: https://datatracker.ietf.org/doc/html/rfc1952 Section 2.3
-    data.len() >= 10 && &data[..3] == b"\x1f\x8b\x08"
+    data.starts_with(GZIP_MAGIC)
```

Comment:
changed this a bit, but it is more consistent with what was done in other places

Reply:
It's better now 👍